tests/Aspire.Hosting.Azure.Tests/AzureServiceBusExtensionsTests.cs

```diff
@@ -215,7 +215,7 @@ public async Task VerifyWaitForOnServiceBusEmulatorBlocksDependentResources()
         await app.StopAsync();
     }
 
-    [Fact(Skip = "Azure ServiceBus emulator is not reliable in CI - https://github.com/dotnet/aspire/issues/7066")]
+    [Fact]
```

Comment:
Is this just temporary? I thought the SB emulator was still not reliable in CI.

Reply:
Right, we reverted that on main in the meantime. Tests passed though in this PR ;)